app/models/authenticator/base.rb

```diff
@@ -309,14 +310,14 @@ def run_task(taskid, status)
     # TODO: Fix this icky select matching with tenancy
     def match_groups(external_group_names)
       return [] if external_group_names.empty?
-      external_group_names = external_group_names.collect(&:downcase)
 
-      internal_groups = MiqGroup.in_my_region.order(:sequence).to_a
+      external_group_names = external_group_names.collect(&:downcase)
+      internal_group_names = MiqGroup.in_my_region.order(:sequence).to_a.map { |g| g.description.downcase }
 
-      external_group_names.each { |g| _log.debug("External Group: #{g}") }
-      internal_groups.each      { |g| _log.debug("Internal Group: #{g.description.downcase}") }
+      _log.debug("External Groups: #{external_group_names.join(", ")}")
+      _log.debug("Internal Groups: #{internal_group_names.join(", ")}")
```

Comment:
I considered dropping this logging entirely, because I don't think it's useful and it's verbose.  Thoughts?

Reply:
haha, I just commented on this... I'm fine with only doing external groups but wonder if that's enough.